### ISSUE TITLE
Slideshow does not wait for video

### ISSUE THREAD

**Issue by ron4mac** [fancyapps/ui#246]
When running the slideshow, items such as video or audio are not allowed to complete before advancing the slideshow.

Note: worked okay in version 3 but version 3 has a bug that it advances the slide for video even when not in slideshow mode.

**Comment by sjd23** [comment]
I'll add this note here since the other issue was closed:

> I found this comment where it was stated that this is not a problem anymore, and the old jsfiddle provided seems to support that idea. This is not the case with 5.0. Is this because a bug was introduced that incidentally removed the support OR was it an intentional feature removal/omission in the later releases?
> 
> "Hi,
> 
> Slideshow now waits for html5 video to end (starting from v3.4.1)"
> 
(from: https://github.com/fancyapps/fancybox/issues/1983#issuecomment-417657084)

**Comment by fancyapps** [comment]
@sjd23 If you have not noticed, this is a completely different project.

**Comment by sjd23** [comment]
@fancyapps  Of course. I didn't read through the commit history but assumed it was initially ported here as the readme of that repository provides this detail:

"This repository has been archived and is now part of https://github.com/fancyapps/ui"

The feature being gone makes sense to me now though. Thank you.
